pyproject.toml

```diff
@@ -4,6 +4,11 @@ requires = ["setuptools >= 50.0",
             "wheel >= 0.34.0"]  # ought to mirror 'requirements/build.txt'
 build-backend = "setuptools.build_meta"
 
+[tool.codespell]
+skip = "*.genx,*.geny,*.png,*egg*,.git,.hypothesis,.nox,.tox,.idea,__pycache__,_build"
+ignore-words-list = """
+circularly"""
```

Comment:
These days, it's recommended to put configuration stuff in `pyproject.toml` instead of `setup.cfg`, so this is to move a little bit more in that direction.